pubspec.yaml

```diff
@@ -27,6 +27,10 @@ dev_dependencies:
 environment:
   sdk: ">=1.21.0 <2.0.0"
```

Comment:
Should we expand this range to include 2.x.x?

Reply:
it'll work with dev branches .. but when dart 2 stable is released .. we'll need to be selective about our SDK ranges.